packages/toolkit/src/view/catalog/components/tabs/UploadExploreTab.tsx

```diff
@@ -209,13 +208,13 @@ export const UploadExploreTab = ({
             setShowFileTooLargeMessage(false);
           }, FILE_ERROR_TIMEOUT);
           break;
-        case "INSUFFICIENT_STORAGE":
-          setIncorrectFileName(file.name);
-          setShowInsufficientStorageMessage(true);
-          insufficientStorageTimeoutRef.current = setTimeout(() => {
-            setShowInsufficientStorageMessage(false);
-          }, FILE_ERROR_TIMEOUT);
-          break;
+        // case "INSUFFICIENT_STORAGE":
+        //   setIncorrectFileName(file.name);
+        //   setShowInsufficientStorageMessage(true);
+        //   insufficientStorageTimeoutRef.current = setTimeout(() => {
+        //     setShowInsufficientStorageMessage(false);
+        //   }, FILE_ERROR_TIMEOUT);
+        //   break;
```

Comment:
@orangecoloured Even we comment this line, the logic will still block user from uploading the file right? I think the product want to completely eliminate this block?